CHANGELOG.rst

```diff
@@ -18,6 +18,15 @@ Fixes
 ^^^^^
 - Clarified Checksum() data for custom checksum function.
 
+v0.3.1
+------
+Features
+^^^^^^^^
+- Add Float primitive
+- Add s_float to push Float primitives to the current block
+- Add an example for s_float/Float usage
+- Add option to encode Floats as IEEE 754 floating point numbers
+
```

Comment:
Please move these features to the `Upcoming` topic. We're not 100% sure what the next release number will be.

Reply:
Sorry, I haven't thought about that.
I have now moved the features to the upcoming topic.